build/teamcity-publish-artifacts.sh

```diff
@@ -22,8 +25,9 @@ if [[ "$TC_BUILD_BRANCH" != *alpha* ]] && [ "$TEAMCITY_BUILDCONF_NAME" == 'Publi
 	    ;;
 	esac
 
-	build/builder.sh make TYPE=$TYPE testbuild TAGS=acceptance PKG=./pkg/acceptance
-	(cd pkg/acceptance && ./acceptance.test -i $image -b /cockroach/cockroach -nodes 4 -test.v -test.timeout -5m)
+        # TODO(bdarnell): This has broken with recent changes to the acceptance tests.
```

Comment:
Extraneous space?

Reply:
Hmm, this file is indented with a mix of tabs and spaces (and my editor only interprets tabs as 2 spaces in go-mode; in other languages it uses python rules in which tabs are always 8 spaces). Untabified the whole file.